### PR TITLE
Adds load timestamp to graph for retrieving resource template by mome…

### LINKED ISSUE #75: refactor: dry up tests by making functions/matchers avail
currently, multiple tests have functions such as `expect_value_in_selector_textContent` added.  It would be nice to dry this up.

possible approaches:

- from https://github.com/jest-community/jest-extended#setup
```
If you are already using another test framework, like jest-chain, then you should create a test setup file and require each of the frameworks you are using.
```
I don't know if we can use node's `require` to load a simple file with functions.
- create a node module, akin to jest-extended (https://github.com/jest-community/jest-extended)

#Requires #74

### PR DESCRIPTION
**Pull request by justinlittman**
…nto.

closes #75

### CODE REVIEW

File: src/reducers/index.js [hunk 1]
```diff
@@ -112,7 +112,9 @@ export const setResourceTemplate = (state, action) => {
     output = refreshResourceTemplate(output, propertyAction)
   })
 
+  console.log(action.payload.loadTimestamp)
```
Comment (justinlittman) [inline src/reducers/index.js]: Cruft.

File: src/GraphBuilder.js [hunk 2]
```diff
@@ -22,10 +22,13 @@ export default class GraphBuilder {
     // A relative URI to use as the base
     const baseURI = rdf.namedNode('')
 
-    // Is there ever more than one base node?
-    Object.keys(this.state).forEach((key) => {
-      this.buildTriplesForNode(baseURI, rdf.namedNode(this.state[key].rdfClass), this.getPredicateList(key))
+    Object.keys(this.state).forEach((rtId) => {
+      // Assuming that loadTimestamp will be close enough for included template
+      // and so only will record for main template.
```
Comment (jermnelson) [inline src/GraphBuilder.js]: Not blocking but what would it take to record this timestamp for all of the embedded resources in the  main template's propertyTemplates?

Comment (justinlittman) [reply src/GraphBuilder.js]: Yes, but going to hold pending #619.

**Comment by justinlittman** [comment]
Closing because going to use resource create timestamp from Trellis.

**Pull request closed without merging**